lib/src/widgets/core/messages_list.dart

```diff
@@ -118,7 +118,6 @@ class _MessagesListState<T extends MessageBase> extends State<MessagesList> {
   /// It will vary depending on the [MessagePosition]
   double _bottomItemPadding(int index, MessagePosition messagePosition) {
     double padding = 8.0;
-    if (index == 0) return 0;
```

Comment:
I understand the padding shift from top level to message level, but why remove this line?